wandb/sdk/artifacts/artifact.py

```diff
@@ -532,6 +532,27 @@ def type(self) -> str:
         """The artifact's type. Common types include `dataset` or `model`."""
         return self._type
 
+    @property
+    def url(self):
+        """
+        Constructs the URL of the artifact.
+
+        Returns:
+            str: The URL of the artifact.
+        """
+        if self.collection.is_sequence():
+            return f"{self._client.app_url}{self._entity}/{self._project}/artifacts/{quote(self._type)}/{quote(self.collection.name)}/{self._version}"
+        else:
+            if self._project.startswith("wandb-registry-"):
```

Comment:
I think you can use the public api method `is_artifact_registry_project()`

Reply:
Yep, added it!